sklearn/linear_model/_glm/glm.py

```diff
@@ -430,6 +430,19 @@ class PoissonRegressor(GeneralizedLinearRegressor):
 
     n_iter_ : int
         Actual number of iterations used in the solver.
+
+    Examples
+    ----------
+    >>> from sklearn import linear_model
+    >>> clf = linear_model.PoissonRegressor(alpha=0.95)
+    >>> X = [[0, 0], [1, 1], [2, 2]]
```

Comment:
Can we have an example where the `coef_` are different between the two features?

Reply:
Good idea.
I fixed it.